### ISSUE TITLE
request: add patcherize functionality to Pd vanilla

### ISSUE THREAD

**Issue by porres** [pure-data/pure-data#1099]
I think it'd be great if we could do it (create abstractions/subpatches) in vanilla, out of the box.

**Comment by umlaeute** [comment]
what's wrong with using an external? apart from it being buggy.


feel free to submit fixes for the remaining issues on https://git.iem.at/pd-gui/punish and i'll see to get it included.

**Comment by porres** [comment]
yeah, way out of my league to find and propose fixes, but hey, let me ask, wasn't this on deken? can't find it anymore and now I tried it and it's really really really bugged and not working at all (I remember I was able to test it reasonably well a while ago).
